### ISSUE TITLE
There are some troubles in your program

### ISSUE THREAD

**Issue by youngGQ** [Anorov/cloudflare-scrape#418]
cfscrape version 2.1.1
url = https://cn.investing.com/markets/united-states
ValueError: Unable to identify Cloudflare IUAM Javascript on website. Cloudflare may have changed their technique, or there may be a bug in the script.

**Comment by SpangleLabs** [comment]
Yup, it's broken, the main developer is not monitoring this, see #406
